app/admin/admin_users.rb

```diff
@@ -0,0 +1,27 @@
+ActiveAdmin.register AdminUser do
+  permit_params :email, :password, :password_confirmation
+
+  index do
+    selectable_column
+    id_column
+    column :email
+    column :current_sign_in_at
+    column :sign_in_count
```

Comment:
I do not thing this columna is util because you do not have it in your model. (Same for all column in this case ;) )

Reply:
I didn't see that. Thanks a lot. 🙂